src/length.cpp

```diff
@@ -1,12 +1,9 @@
 #include "r_trie.h"
 
 template <typename T>
-int radix_len(SEXP radix){
+static inline int radix_len(SEXP radix){
```

Comment:
tldr is `static inline` helps keep size down. We're already pushing up against a 5 MiB `libs/` in the installation, generating an `R CMD check` NOTE. I actually had to do some voodoo to the internals to get rid of the NOTE.

Everybody thinks `inline` is an instruction for the compiler, but it's really more for the linker. This is all assuming that `static inline` works reasonably similar in C++ to how it does in C, which I believe is the case.